apps/pollbook/backend/src/backup_worker.ts

```diff
@@ -20,7 +20,7 @@ import {
 } from './voter_checklist';
 import { LocalStore } from './local_store';
 
-const BACKUP_INTERVAL = 1_000 * 60; // 1 minute
+const BACKUP_INTERVAL = 1_000 * 10; // 1 minute
```

Comment:
is this intentional?

Reply:
no was just making it faster for testing will revert thanks